src/devices/sara_u280.c

```diff
@@ -54,9 +56,11 @@ static bool sara_u280_get_subscriber_number(struct serial_buffer *sb,
         bool status = false;
         for (size_t i = 0; i < SARA_U280_SUBSCRIBER_NUMBER_RETRIES && !status; ++i)
                 status = gsm_get_subscriber_number(sb, ci);
+                if (!status)
+                        delayMs(FAILED_READ_BACKOFF_MS);
 
         if (!status)
-                pr_warning("[cell] Failed to read phone number\r\n");
+                pr_warning("[cell] Phone number not available\r\n");
```

Comment:
So what do the docs say about this? Is there no way to differentiate between it not being available vs a failure?

Reply:
Internet research says that the module may not return a phone number when asked, and this is a valid case. i.e. it can return just "OK" with no data.  This change acknowledges this possibility.